### ISSUE TITLE
The Migration/DepartmentName cop doesn't handle comments with double comments

### ISSUE THREAD

**Issue by tas50** [rubocop-hq/rubocop#7620]
# Expected behavior

The `Migration/DepartmentName` cop shouldn't trigger on a comment like this:

```
# rubocop:disable Style/SafeNavigation # support Ruby < 2.3.0
```

I understand this might not be considered an appropriate RuboCop comment. It's not code within my codebase so I don't really control how the user here decided to write their comments. Regardless of the syntax it seems incorrect to alert the `Migration/DepartmentName` cop for this comment line and worse that the cop claims it autocorrect it every time RuboCop runs.

## Actual behavior

```
foo/libraries/bar/string_utils.rb:46:54: C: [Corrected] Migration/DepartmentName: Department name is missing.
    # rubocop:disable Style/SafeNavigation # support Ruby < 2.3.0
                                                     ^^^^
```

## Steps to reproduce the problem

Run `rubocop -a --only Migration/DepartmentName` against a file with the above comment.

## RuboCop version

```
0.78.0
```
